bindings_ffi/README.md

```diff
@@ -25,6 +25,18 @@ The generated bindings (`xmtp_dh.kt`) and the cross-compiled binaries (`jniLibs`
 
 `Cross` allows us to run cross-platform builds without needing to download all of the relevant toolchains and should work regardless of your host OS. It is possible that the build time can be improved by building natively without Cross.
 
+# Running tests
+
+Ensure a local API host is running - run `dev/up` from the repo root.
+
+You'll need to do the following one-time setup to run Kotlin tests:
+
+- Run `brew install kotlin` to get `kotlinc`
+- Install the JRE from `www.java.com`
+- Run `make install-jar` and add both jars to your CLASSPATH environment variable, for example add `export CLASSPATH=$HOME/jna/jna.jar:$HOME/jna/kotlinx-coroutines-core-jvm.jar` to your `.zshrc`
+
+If you want to skip the setup, you can also run `cargo test -- --skip kts` to only run Rust unit tests. We are currently skipping this in CI.
+
```

Comment:
This seems reasonable to me given `bindings_ffi` is exlcuded from the workspace.